test/e2e/config/e2e_conf.yaml

```diff
@@ -17,85 +17,96 @@ providers:
 - name: cluster-api
   type: CoreProvider
   versions:
-  - name: ${CAPI_FROM_RELEASE}
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CAPI_FROM_RELEASE}/core-components.yaml"
+  - name: v1.6.1
+    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.1/core-components.yaml"
     type: "url"
-    contract: ${CONTRACT_FROM}
+    contract: v1beta1
     replacements:
     - old: --metrics-addr=127.0.0.1:8080
       new: --metrics-addr=:8080
     files:
-    - sourcePath: "../data/shared/${CAPI_FROM_RELEASE:0:4}/metadata.yaml"
-  - name: ${CAPI_TO_RELEASE}
-    # Use manifest from source files
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CAPI_TO_RELEASE}/core-components.yaml"
+    - sourcePath: "../data/shared/v1.6/metadata.yaml"
+  - name: v1.5.5
+    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.5.5/core-components.yaml"
     type: "url"
-    contract: ${CONTRACT_TO}
-    files:
-    - sourcePath: "../data/shared/${CAPI_TO_RELEASE:0:4}/metadata.yaml"
+    contract: v1beta1
     replacements:
-    - old: "--leader-elect"
-      new: "--leader-elect=false"
+    - old: --metrics-addr=127.0.0.1:8080
+      new: --metrics-addr=:8080
+    files:
+    - sourcePath: "../data/shared/v1.5/metadata.yaml"
 - name: kubeadm
   type: BootstrapProvider
   versions:
-  - name: ${CAPI_FROM_RELEASE} # latest published release in the v1alpha4 series; this is used for v1alpha4 --> v1beta1 clusterctl upgrades test only.
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/${CAPI_FROM_RELEASE}/bootstrap-components.yaml"
+  - name: v1.6.1 # latest published release in the v1alpha4 series; this is used for v1alpha4 --> v1beta1 clusterctl upgrades test only.
```

Comment:
Same should it be 1.6.3, also I am removing the comment in my PR

Reply:
fixed